src/http-servers/plain-text-server.js

```diff
@@ -0,0 +1,19 @@
+import http from 'http';
+
+const port = 3000;
+
+const requestHandler = (request, response) => {
```

Comment:
I see that in 3 places you have the same code for instantiating the server differencing only on content type and what data to serve. I suggest you to extract all code from this file to a separate util function called `customServer(contentType, content)` and import it to html, json & plain-text servers & call it like `customServer('text/plain', 'Hello World')`